model/filing.py

```diff
@@ -0,0 +1,25 @@
+""" Filings """
+from .base import BaseModel
+
+class FilingCollection(BaseModel):
```

Comment:
Why not call this Filings?

Reply:
I think I originally had a data class for the individual filings, elections, etc. and I thought it was confusing to have Filing and Filings because the names are too similar. I might still go that way. I'm not sure about the performance differences between instantiating an object and a dict. I'm not wed to the `...Collection` names, though.